paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/ElementsSessionRepository.kt

```diff
@@ -86,6 +95,15 @@ internal class RealElementsSessionRepository @Inject constructor(
     }
 }
 
+private fun StripeIntent.withoutWeChatPay(): StripeIntent {
```

Comment:
This is more scary to me the more I read it. It's easy to miss this. Is there another way to filter this out?

Reply:
I don’t think there is. We don’t get the server-side filtering if we fall back to the raw API 😕